CMSgov/blueprint_api
# setting ociso and aws for new projects

*What does this PR do?*
on project creation attaches inherited components 
*Jira ticket number?*
[ISPGBSS-1139](https://jiraent.cms.gov/browse/ISPGBSS-1139)

*Changes*

- on project create adds ociso and aws components 

*Testing*

- create a project 
- check the components attached to the project
